### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TommyZihao**
Add Chinese Colab Tutorial

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 embedchain is a framework to easily create LLM powered bots over any dataset. If you want a javascript version, check out [embedchain-js](https://github.com/embedchain/embedchainjs)
 
+Chinese Colab Tutorial:https://colab.research.google.com/drive/10_7Y0x4YXWVjuhhYwVraGQLpKAatTQTm?usp=sharing
```
Comment (taranjeet) [inline README.md:10]: Hey that's a nice initiative. thanks.
One request: can you please create a section called Colab Notebooks just before Advanced section
and then list yours there?

Comment (TommyZihao) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -267,6 +267,12 @@ _The embedding is confirmed to work as expected. It returns the right document,
 
 **The dry run will still consume tokens to embed your query, but it is only ~1/15 of the prompt.**
 
+## Colab Notebook and Video Tutorials
+
+Chinese Colab Tutorial:https://colab.research.google.com/drive/10_7Y0x4YXWVjuhhYwVraGQLpKAatTQTm?usp=sharing
+
+Chinese Video Tutorial:https://www.bilibili.com/video/BV1YX4y1H7oN
```
Comment (taranjeet) [inline README.md:274]: Hey, one small request.

can you please add your video tutorial in this repo? https://github.com/embedchain/community-showcase

Idea is to have a community showcase repo where all the tutorials can be clubbed together.

Comment (TommyZihao) [reply README.md:274]: sure

**Pull request merged**